pages/abu-dhabi.js

```diff
@@ -0,0 +1,844 @@
+              filter: 'drop-shadow(5px 5px 5px #000000AA)',
+              position: 'relative',
+              zIndex: 20
+            }}
+          >
+            <Heading
+              as="h2"
+              sx={{
+                fontFamily: 'moonblossom',
+                textAlign: 'center',
+                margin: '8%'
+              }}
+            >
+              Build stupid s#!t, get stupid prizes.
+            </Heading>
+          </Box>
+          <Box
+            sx={{
+              zIndex: 1,
+              width: 'max-content',
+              backgroundImage: "url('/elements/yellow-strip@stretch.svg')",
+              backgroundRepeat: 'no-repeat',
+              backgroundSize: '100% 100%',
+              width: '75%',
+              position: 'relative',
+              zIndex: 30,
+              top: '-15%',
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center',
+              py: '3%',
+              filter: 'drop-shadow(5px 5px 5px #00000099)'
+            }}
+          >
+            <Heading
+              as="h2"
+              sx={{
+                fontFamily: 'p22-stanyan',
+                mx: '8%',
+                p: 0,
+                wordBreak: 'keep-all',
+                whiteSpace: 'nowrap',
+                width: 'max-content',
+                fontSize: ['1.2em', '1.4em']
+              }}
+            >
+              {/* TODO: Change [EXAMPLECITY] to your event's city */}
+              Abu Dhabi - March&nbsp;15-16
+            </Heading>
+          </Box>
+        </Box>
+        <Box
+          sx={{
+            position: 'absolute',
+            width: '100%',
+            height: '100vw'
+          }}
+        >
+          <Image
+            sx={{
+              width: '10%',
+              height: '10%',
+              zIndex: 2,
+              position: 'absolute',
+              top: '50%',
+              left: '10%'
+            }}
+            src="/elements/stars/blue.png"
+            alt="Blue paper star"
+          />
+          <Image
+            sx={{
+              width: '10%',
+              height: '10%',
+              zIndex: 2,
+              position: 'absolute',
+              top: '55%',
+              right: '15%'
+            }}
+            src="/elements/stars/yellow.png"
+            alt="Yellow paper star"
+          />
+          <Image
+            sx={{
+              width: '10%',
+              height: '10%',
+              zIndex: 2,
+              position: 'absolute',
+              top: '70%',
+              left: '20%',
+              transform: 'rotate(180deg)'
+            }}
+            src="/elements/stars/pink.png"
+            alt="Pink paper star"
+          />
+        </Box>
+        <Link
+          // TODO: Change [SLUG] to your event's slug (lowercase, dashed version of your event name), such as san-francisco for Scrapyard San Francisco
+          href="https://hackclub.tech/scrapyard"
```

Comment:
In both instances (Line 230 and Line 765), can you use our Hack Club form URL rather than using ``hackclub.tech``? I know it currently redirects to the same URL, but we would prefer if the form is directly linked so the redirect can't be changed without a PR.